src/js/electron/ipc/windows/messages.ts

```diff
@@ -10,6 +10,9 @@ import {
 } from "../types"
 
 export type NewTabSearchParams = {
+  host: string
```

Comment:
These are for when we initiate a search from the detail window (possible from field context menus). When there are no existing search windows to create a new tab on, we already create a new window but now we also pass in the detail window's `host` and `port` as query args so that the search window's initialization can use these instead of defaulting to localhost:9867 for adding a cluster. `connId` will now also be sent along through ipc to the new/existing window much like spaceId already is.

Reply:
Awesome